src/components/elements/dropdown.tsx

```diff
@@ -124,7 +136,7 @@ const ListContainer = styled.li`
     list-style: none;
 `
 
-const ListItem = styled.li`
+const ListItem = styled.li<any>`
     color: var(--color-grey-6);
```

Comment:
<img width="603" alt="Screenshot 2022-08-03 at 08 11 24" src="https://user-images.githubusercontent.com/103182473/182590482-b2b9d1bc-f7bf-4d0e-a742-2a4a962a92f9.png">

Reply:
@NikitK-deriv which error do you get here?

Reply:
Resolved 🙇